src/modules/fw_wind_estimator/module.yaml

```
@@ -0,0 +1,51 @@
+module_name: fw_wind_estimator
+parameters:
+- group: FW Wind Estimator
```

```suggestion
- group: 3D Wind Estimator
```
To clearly distinguish it from the existing (2D) wind estimators?